docs/archapi.md

```diff
@@ -463,21 +471,66 @@ Cell Delay Methods
 Returns the delay for the specified path through a cell in the `&delay` argument. The method returns
 false if there is no timing relationship from `fromPort` to `toPort`.
 
-### TimingPortClass getPortTimingClass(const CellInfo *cell, IdString port, int &clockInfoCount) const
+### TimingPortClass getPortTimingClass(const CellInfo \*cell, IdString port, int &clockInfoCount) const
 
 Return the _timing port class_ of a port. This can be a register or combinational input or output; clock input or
 output; general startpoint or endpoint; or a port ignored for timing purposes. For register ports, clockInfoCount is set
 to the number of associated _clock edges_ that can be queried by getPortClockingInfo.
 
-### TimingClockingInfo getPortClockingInfo(const CellInfo *cell, IdString port, int index) const
+### TimingClockingInfo getPortClockingInfo(const CellInfo \*cell, IdString port, int index) const
 
 Return the _clocking info_ (including port name of clock, clock polarity and setup/hold/clock-to-out times) of a
 port. Where ports have more than one clock edge associated with them (such as DDR outputs), `index` can be used to obtain
 information for all edges. `index` must be in [0, clockInfoCount), behaviour is undefined otherwise.
 
+Partition Methods
+-----------------
+
+Partitions are subsets of BelIds and cell types used by analytic placer to
+seperate types of BELs during placement.  Typical partitions are:
+ - All LUT BELs
```

Comment:
ime HeAP has worked better with LUTs and FFs in the same partition - this may be more of an implementation detail than a general truth though

Reply:
That may be true, but I think this is a problem with the documentation, rather than the implementation.  The CutSpreader needs occupancy / bel counts, and this is definitely based on effectively interchangeable locations rather than LUT+FF.

I'll try to reword this to clarify.

Reply:
I've tried to reword this.